### PR TITLE
[RU] Translation report

### PR DESCRIPTION
**Pull request by nscyclone**

### CODE REVIEW

File: _data/locales/ru.yml [hunk 1]
```diff
@@ -10,22 +10,22 @@ sections:
...
 translation_report:
-  section: Section
-  original_title: Original Title
-  translation_title: Translation Title
-  original_version: Original Version
-  translation_version: Translation Version
-  version_severity: Version Severity
+  section: Раздел
+  original_title: Наименование оригинала
+  translation_title: Наименование перевода
```
Comment (brain-geek) [inline _data/locales/ru.yml]: `Заголовок перевода`? Same one line above

File: _data/locales/ru.yml [hunk 2]
```diff
@@ -10,22 +10,22 @@ sections:
...
+  original_version: Версия оригинала
+  translation_version: Версия перевода
+  version_severity: Степень различий в версиях
```
Comment (brain-geek) [inline _data/locales/ru.yml]: Sounds robotic. How about `Разница в версиях`?

File: _data/locales/ru.yml [hunk 3]
```diff
@@ -10,22 +10,22 @@ sections:
...
+  translation_report: Состояние перевода
 
 version_messages:
   outdated: "Некоторое содержимое этого урока может оказаться устаревшим."
   major: "С момента последнего обновления перевода в оригинальный урок были внесены серьёзные изменения."
   minor: "С момента последнего обновления перевода в оригинальный урок были внесены небольшие изменения."
   patch: "С момента последнего обновления перевода в оригинальный урок были внесены небольшие правки."
   none: "Это перевод актуальной версии оригинального урока."
-  error: "There is an error in the version definition of this chapter."
+  error: "В объявлении версии этого урока обнаружена ошибка."
```
[the remark this answers is not in the document]
Comment (nscyclone) [reply _data/locales/ru.yml]: Do you mean `в объявлении версии` or just `в объявлении`?

[the remark this answers is not in the document]
Comment (brain-geek) [reply _data/locales/ru.yml]: The latter one. We need to specify that it's some internal error not related to lesson contents. IMO, that would sound better.

**Comment by doomspork** [comment]
@nscyclone / @brain-geek feel free to merge when you're happy 👍

**Pull request merged**